wolfcrypt/benchmark/benchmark.c

```diff
@@ -3989,15 +3989,32 @@ static void bench_rsa_helper(int doAsync, RsaKey rsaKey[BENCH_MAX_PENDING],
 #ifndef WOLFSSL_RSA_VERIFY_ONLY
     DECLARE_VAR_INIT(message, byte, len, messageStr, HEAP_HINT);
 #endif
-
-    DECLARE_ARRAY_DYNAMIC(enc, byte, BENCH_MAX_PENDING, rsaKeySz, HEAP_HINT);
+    #if !defined(WOLFSSL_MDK5_COMPLv5)
+	  /* MDK5 compiler regard this as a executable statement, and does not allow declarations after the line. */
+        DECLARE_ARRAY_DYNAMIC_DEC(enc, byte, BENCH_MAX_PENDING, rsaKeySz, HEAP_HINT);
+	  #else
```

Comment:
It looks like there may be some hard tabs around this area.  Could you convert them to 4 spaces to help with readability?  Thanks!

Reply:
removed